apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -237,6 +240,72 @@ const uploadControlCss = css`
   }
 `
 
+const inputControlCss = css`
+  .ant-input.white {
+    background: white;
+    &:focus, &:active {
+      background: white !important;
+    }
+  }
+`
+
+const commentCss = css`
+  .ant-comment {
```

Comment:
is it possible not to use global styles? you can wrap a component with style overrides.

Reply:
Moved into component scope in `b3768a18`